src/sound/lomse_score_player.cpp

```diff
@@ -249,18 +242,7 @@ void ScorePlayer::stop()
     if (m_pThread)
     {
         m_fShouldStop = true;
-
-        //wait 500 ms for termination
-        if(!m_pThread->timed_join(boost::posix_time::milliseconds(500)))
```

Comment:
This delay of 500ms hid the problem now detected. In my app. the use can have several scores open in different tabs. If one of the is being played back and the user switches to another tab, the score must stop to play. Unfortunately, it also sends repaint events for removing the higlight, and these events conflicts with the paint events for the new selected tab. The problem was masked by this delay but has nothing to do with this PR. So this PR looks good. I'm going to some additional tests.